File: JWDataTracker.Domain/Entity/Publisher.cs
using System;
using System.Collections.Generic;

namespace JWDataTracker.Infrastructure
{
    public partial class Publisher
    {
        public long PublisherId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string CreatedDate { get; set; }
        public long? CongregationId { get; set; }
        public long IsElder { get; set; }
        public long IsMs { get; set; }
        public long IsRp { get; set; }
        public long IsUnBaptized { get; set; }
        public long GroupNumber { get; set; }
        public long IsBaptized { get; set; }

        public virtual Congregation Congregation { get; set; }
    }
}

File: JWDataTrackerWeb/src/app/shared/models/publisher.ts
import { LookUp } from "./lookup";
import { MidWeekScheduleItem } from "./midWeekSchedule";

export class Publisher {
    publisherId: number;
    firstName: string;
    lastName: string;
    congregationId:number;
    isElder: boolean;
    isMs: boolean;
    isRp: boolean;
    isUnBaptized: boolean;
    groupNumber:number;
    fullName:string;
    mostRecentPart: RecentPart;
    recentParts: RecentPart[];

    privilege: number;
    status: number;
    gender: number;

    privilegeLookUp: LookUp;
    statusLookUp: LookUp;
    genderLookUp: LookUp;
}

export class RecentPart{
    date: Date;
    category: string;
    role: string;
    isPartner: Boolean;
    scheduledDateDT: Date;
    part: string;
}